### ISSUE TITLE
Databricks notebook task not found

### ISSUE THREAD

**Issue by isantillan1** [microsoft/Purview-ADB-Lineage-Solution-Accelerator#13]
**Describe the bug**
Databricks notebook task not found

**To Reproduce**
1. run-through [Purview-ADB-Lineage-Solution-Accelerator](https://github.com/microsoft/Purview-ADB-Lineage-Solution-Accelerator)
2. Databricks objects, look for Databricks Notebook Tasks

**Expected behavior**
Expecting 67 objects loaded: 
![image](https://github.com/microsoft/Purview-ADB-Lineage-Solution-Accelerator/raw/main/assets/img/readme/browse_assets.png)
Only finding 3:
![image](https://i.imgur.com/9rr1wOi.png)

Expecting Databricks notebook tasks:
![image](https://github.com/microsoft/Purview-ADB-Lineage-Solution-Accelerator/raw/main/assets/img/readme/databricks_task_related.png)

Unable to find:
![image](https://i.imgur.com/K5tYWvX.png)
![image](https://i.imgur.com/dk8upD7.png)

**Logs**
1. Please include any Spark code being ran that generates this error
2. Please [include a gist](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) to the OpenLineageIn and PurviewOut logs
3. See how to stream [Azure Function Logs](https://docs.microsoft.com/en-us/azure/azure-functions/streaming-logs)

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, Mac]
 - OpenLineage Version: [e.g. name of jar]
 - Databricks Runtime Version: [e.g. 6.4, 9.1, 10.1]
 - Cluster Type: [e.g. Job, Interactive]
 - Cluster Mode: [e.g. Standard, High Concurrency, Single]
 - Using Credential Passthrough: [e.g. Yes, No]

**Additional context**

**Comment by isantillan1** [comment]
1. ![image](https://i.imgur.com/FdBV9Sc.png)
2. ![image](https://i.imgur.com/aHdlwDH.png)
3. ![image](https://i.imgur.com/jexy0O2.png)
4.  ![image](https://i.imgur.com/y7Apiwv.png)

**Comment by wjohnson** [comment]
@isantillan1 Thank you so much for going through the solution accelerator!  To generate this situation, did you run the notebook sample provided in the Demo deployment?

If so, you will only see the `Notebook` and it's `Databricks Process` child(ren) and not at `Notebook Task`.  `Notebook Task` represents a Databricks Notebook job / task as created through the Jobs API.

Does that fit your scenario?

**Comment by isantillan1** [comment]
Yes, ran through the 1st step in the [Using the Connector](https://github.com/microsoft/Purview-ADB-Lineage-Solution-Accelerator/#using-the-connector) to "Run all".

The following steps specifically asks to look for _"Notebook Tasks"_ which is not found. 

> Click on the "Databricks" section, then click on the "Databricks Notebook" tile which corresponds to the notebook you ran. In the Properties _or_ Related tabs select one of the _"Notebook Tasks"_ which represent a task in a Databricks job.

**Comment by wjohnson** [comment]
This is great feedback as well.  @mattsavarino and I will coordinate on addressing this in the README for our next week's release.
